fontir/src/variations.rs

```diff
@@ -21,6 +21,44 @@ use write_fonts::{
 
 use crate::error::VariationModelError;
 
+/// Trait for performing banker's rounding on a value.
+///
+/// Banker's rounding rounds half to even. This means that half-way values
+/// are rounded to the nearest even number. For example, 2.5 rounds to 2.0,
+/// 3.5 rounds to 4.0, and -2.5 rounds to -2.0.
+///
+/// This is the same rounding mode as [`f64::round_ties_even`], which was
+/// stabilized in Rust 1.77.0. It also matches Python's buit-in `round` function.
+/// It is usually preferred over alternative approaches because it reduces bias
+/// and errors in calculations.
+///
+/// This trait provides a generic way to apply banker's rounding to different types,
+/// such as `f64` and `kurbo::Vec2`.
+pub trait BankerRound<U, T = Self> {
```

Comment:
Suggest naming to match the rust method, e.g. RoundTiesEven not "BankerRound." I notice the documentation for round ties even doesn't even mention bankers, it's not key to the concept.

Reply:
Also someone familiar with the Rust library are far more likely to recognize round_ties_even

Reply:
I thought I could not shadow the built-in f64::round_ties_even method with my own trait, but turns out it works.. I'll rename